features/step_definitions/et1_ccd_steps.rb

```diff
@@ -261,9 +261,12 @@
 end
 
 And(/^the CCD claim should have (\d+) ACAS certificates$/) do |number|
-  sleep 10
   office = @respondent[0]["expected_office"]
   ccd_office_lookup = ::EtFullSystem::Test::CcdOfficeLookUp
   ccd_object = EtFullSystem::Test::Ccd::Et1CcdSingleClaimant.find_by_reference(@claim_reference, ccd_office_lookup.office_lookup[office][:single][:case_type_id])
-  ccd_object.assert_acas_pdf_file_quantity(5)
+  expected_names = []
```

Comment:
Can you do

```ruby

expected_names = @respondent[0..4].map { |r| "acas_#{r[:name]}.pdf" }
```

i.e. grab up to the first 5 respondents (I know the instance variable is called respondent - but as you know, it is an array) - map each one to a string